Update footer and fix code

mohamdlog opened the pull request.

git-voo reviewed (commented):
Hey @mohamdlog, I was taking a look at your PR but I'm not sure what I'm looking out for. Seems like you replaced the social icons on the footer and reformatted the routes in `app.js`?
A little comment would help, please.

mohamdlog commented:
You are correct. Tt seems while we were trying to fix the code Monday, one of the social icons was replaced. This simply reverts that change. As for the `app.js`, I just cleaned up the indentation and removed an unused import. I'll provide detailed descriptions next time.

git-voo commented:
> You are correct. It seems while we were trying to fix the code Tuesday, one of the social icons was replaced. This simply reverts that change. As for the `app.js`, I just cleaned up the indentation and removed an unused import. I will provide more detailed descriptions next time.

Great! okay then.
I asked because I thought it was the layout implementation that we discussed that you pushed. Thanks for clarifying. I'd merge your changes now and create a card for the layout implementation on Trello for next time
